hm-service-account-api-rest-v1.yml

```diff
@@ -294,6 +294,43 @@ paths:
           description: A Service Account Token associated to a fleet application
       operationId: ApiWeb.FleetImportController.index
       description: Get the status of VINs that have previously been registered for data access clearance.
+  /v1/fleets/{vin}/cancel_activation:
+    post:
+      security:
+        - ServiceAccountToken: []
+      tags:
+        - Fleet
+      responses:
+        '200':
+          schema:
+            $ref: '#/definitions/FleetVin'
+          description: Vehicle will be canceled soon
```

Comment:
Proposal: "Vehicle activation is being canceled"

Reply:
Done